Automattic/jetpack
# Unit tests: Update @covers annotations

#### Changes proposed in this Pull Request:

* fix incorrectly defined @covers annotations

Also, I updated some of the annotations to use class scope instead of method scope as recommended in PHPUnit docs: https://phpunit.readthedocs.io/en/8.4/annotations.html#appendixes-annotations-covers-tables-annotations

#### Testing instructions:






* Check Code coverage build in CI
* There should be no warnings related to @covers annotations such as:
`Trying to @cover or @use not existing class or interface "Jetpack->wp_rest_authenticate".`

#### Proposed changelog entry for your changes:

* n/a
